extensions/cl_intel_unified_shared_memory.asciidoc

```diff
@@ -721,7 +722,11 @@ Otherwise, it will return one of the following error values:
       | Returns allocation flags for the Unified Shared Memory allocation.
 
         Returns `0` if no allocation flags were specified for the Unified Shared Memory allocation and for `CL_MEM_TYPE_UNKNOWN_INTEL` allocations.
+| `CL_MEM_ALLOC_BUFFER_LOCATION_INTEL`
+  | cl_uint
+      | Returns buffer location for the Unified Shared Memory allocation.
 
+        Return the ID of default global memory if no buffer location property were specified.
```

Comment:
This description should also move out of the USM spec and into the `cl_intel_mem_alloc_buffer_location` extension spec.

Reply:
I see that this section has been moved to the `cl_intel_mem_alloc_buffer_location` spec but it hasn't been removed from the USM spec.